build/build-treemap.js

```diff
@@ -8,6 +8,33 @@
 const fs = require('fs');
 const GhPagesApp = require('./gh-pages-app.js');
 
+/**
+ * Extract only the strings needed for lighthouse-treemap into
+ * a script that sets a global variable `locales`, whose keys
+ * are locale codes (en-US, es, etc.)
+ */
+function buildLocales() {
+  const locales = require('../lighthouse-core/lib/i18n/locales.js');
+  const clonedLocales = JSON.parse(JSON.stringify(locales));
+
+  for (const lhlMessages of Object.values(clonedLocales)) {
+    for (const icuMessageId of Object.keys(lhlMessages)) {
+      const lhlMessage = lhlMessages[icuMessageId];
+      delete lhlMessages[icuMessageId];
```

Comment:
ok, c'mon y'all now it's *really* not a clone 😆 still no construction love?

Reply:
lol ok i surrender

Reply:
also renamed this to `strings` since it is different from the real locales